execution/aggregate/accumulator.go

```diff
@@ -140,9 +137,6 @@ func newGroupAcc() *genericAcc {
 }
 
 func (g *genericAcc) Add(v float64, _ *histogram.FloatHistogram) {
-	if g.skipNaNs && math.IsNaN(v) {
```

Comment:
Hm won't this include NaNs in max/min calculation?

Reply:
Dont we have tests for that? If not, let me add some, i trusted that our tests would catch any issues there

Reply:
at least in the newly added test promql returns NaN in a min aggregation but the engine returns empty result

Reply:
```
		{
			name: "fuzz - min with NaN",
			load: `load 30s
              http_requests_total{pod="nginx-1", route="/"} 124.00+1.00x40
              http_requests_total{pod="nginx-2", route="/"}  0+0.29x40`,
			query: "min by (route, pod) (sqrt(-http_requests_total))",
		},
```
This one returns NaN in prometheus but nothing in the engine

Reply:
Ah I see. I wonder how much of this behavior is desired vs accidental even in Prometheus 🤷‍♂️

Reply:
Btw what is the result of `max`? Is it still NaN?

Reply:
Ill add same test for max!

Reply:
verified that it works for max too